pomm-project/pomm-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "alias": "pomm",
+    "bundles": {
+        "PommProject\\PommBundle\\PommBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/"
+    },
+    "env": {
+        "DATABASE_DSN": "pgsql://user:pass@127.0.0.1:5432/my_db_name"
```

Comment:
In Doctrine's recipe we use the `DATABASE_URL` env var. See https://github.com/symfony/recipes/blob/master/doctrine/doctrine-bundle/1.6/manifest.json  Do you need a different env var from Doctrine or can you reuse it?